fdbctl/include/fdbctl/ControlCommands.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * ControlCommands.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2025 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifdef FLOW_GRPC_ENABLED
+#ifndef FDB_CLI_LIB_FDB_COMMANDS_H
```

Comment:
Don't want to make this define FDB_CLI_LIB_CONTROL_COMMANDS ? (Add in CONTROL_ and remove double FDB_)